Sources/AllOfSchema.swift

```diff
@@ -12,14 +20,19 @@ public struct AllOfSchemaBuilder {
     let metadata: MetadataBuilder
     let schemaBuilders: [SchemaBuilder]
     
+    /// See AllOfSchema.abstract
```

Comment:
The builders don't need comments on their properties because they shouldn't be public (unless you think their purpose isn't fully explained by the comment on the building).